common/Perf/Azure.Test.Perf/BatchPerfTest.cs

```diff
@@ -48,9 +48,10 @@ public BatchPerfTest(TOptions options) : base(options)
                 }
                 else
                 {
+#if NETFRAMEWORK
```

Comment:
[SYSLIB0014: WebRequest, HttpWebRequest, ServicePoint, WebClient are obsolete](https://learn.microsoft.com/en-us/dotnet/fundamentals/syslib-diagnostics/syslib0014) in .NET6+ and doesn't affect HttpClient. HttpWebRequestTransport is only used with .NET Framework.